### PR TITLE
try.html: Make shield label also clickable

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

**Comment by espadrine** [comment]
Since 648bef473bb5657e5a54, this requires a small rebase.

File: try.html [hunk 1]
```diff
@@ -1126,6 +1126,7 @@ <h2 id="contributors"> Contributors </h2>
     if (target) {
       target.addEventListener('click', makeMarkupDialogListener(tr));
     }
+    tr.querySelector('th').addEventListener('click', makeMarkupDialogListener(tr));
```
Comment (espadrine) [inline try.html]: I recommend setting `tr.querySelector('th')` to a variable with an explicit name.

Comment (techtonik) [inline try.html]: Done.

**Comment by techtonik** [comment]
It looks like the whole <tr> element can be assigned both style and click handler.

**Comment by techtonik** [comment]
@espadrine should be okay now. PTAL.

**Comment by espadrine** [comment]
Thanks for the contribution!

**Comment by techtonik** [comment]
No problemo. :+1:

**Pull request merged**